### PR TITLE
fixes#4386 chore(docs): Document how to see visualization results locally

### LINKED ISSUE #4386: Document how to see visualization results locally
It's not clear how to see the Results page locally from the docs.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-815)
┆Issue Number: EXP-815

### PR DESCRIPTION
**Pull request by LZoog**
fixes #4386 

We were missing docs around this and seeing the Results page locally is definitely handy ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Experimenter is a platform for managing experiments in [Mozilla Firefox](https:/
 
 1.  Install [docker](https://www.docker.com/) on your machine
 
-  - On linux, [setup docker to run as non-root](https://docs.docker.com/engine/security/rootless/)
+- On linux, [setup docker to run as non-root](https://docs.docker.com/engine/security/rootless/)
```
Comment (LZoog) [inline README.md:29]: The unrelated changes were prettier's doing.

File: app/experimenter/nimbus-ui/README.md [hunk 2]
```diff
@@ -185,6 +185,16 @@ if (message !== "success") {
...
+## Access Visualization Data Locally
+
+The Results page renders experiment analysis data and can only be accessed when an experiment is complete, the analysis feature flag is on, and the analysis data returned from the visualization API endpoint is available.
+
+To see the Results page and data locally:
+
+1. Ensure your [Google Credentials](https://github.com/mozilla/experimenter#google-credentials) are configured
+2. Set `FEATURE_ANALYSIS=true` in your local `.env` file
+3. Create a Nimbus Experiment at `localhost/admin` with status "complete," a branch named "control", and another branch named "treatment"
+
```
Comment (jodyheavener) [inline app/experimenter/nimbus-ui/README.md:197]: Do we want to include one of the dummy slugs as well?

**Review by jodyheavener — APPROVED** [review]
One Q makes sense to me tho

File: app/experimenter/nimbus-ui/README.md (hunk 2 above)
Comment (LZoog) [reply app/experimenter/nimbus-ui/README.md:197]: Ah oops you're right. I didn't realize it has to be that specific slug but it does.

**Review by emtwo — APPROVED** [review]
lgtm, thanks for updating the docs!
